textworld/core.py

```diff
@@ -141,29 +141,25 @@ class Environment:
     Here's a minimal example of how to interact with an `Environment`
 
     >>> import textworld
-    >>> game_file, _ = textworld.make(seed=1234)  # Generate a random game.
+    >>> options = textworld.GameOptions()
+    >>> options.seeds = 12345
+    >>> game_file, _ = textworld.make(options, path='./')  # Generate a random game.
     >>> env = textworld.start(game_file)  # Load the game.
     >>> game_state = env.reset()  # Start a new game.
     >>> env.render()
-    I hope your ready to go into rooms and interact with objects, because you've
-    just entered TextWorld! Here is your task for today. First of all, recover the
-    keycard from the shelf. After that, sit the keycard on the rack. Got that? Good!
+    Get ready to pick stuff up and put it in places, because you've just entered
```

Comment:
Does this look ok?

```
>>> import textworld
>>> options = textworld.GameOptions()
>>> options.seeds = 1234
>>> options.nb_objects=5
>>> options.quest_length=2
>>> game_file, _ = textworld.make(options, path='./')
>>> env = textworld.start(game_file)
>>> game_state = env.reset()
>>> env.render()



I hope you're ready to go into rooms and interact with objects, because you've
just entered TextWorld! Here is how to play! First thing I need you to do is to
ensure that the type G chest is open. And then, pick up the keycard from the
type G chest inside the attic. Got that? Good!

-= Attic =-
You arrive in an attic. A normal kind of place. You begin to take stock of
what's in the room.

You make out a type G chest. You can see a TextWorld style locker. The TextWorld
style locker contains a frisbee and a sock.



There is a TextWorld style key on the floor.

>>> command = "take key"
>>> game_state, reward, done = env.step(command)
>>> env.render()
(the TextWorld style key)
You pick up the TextWorld style key from the ground.
```

Reply:
Yes. Seems good.